### PR TITLE
Add page

### PR DESCRIPTION
**Pull request by Alex56dev**

### CODE REVIEW

**Comment by Alex56dev** [comment]
Add

File: _pages/alex56dev.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+ githubHandle: YOUR-USERNAME
+ pageUrl: YOUR-USERNAME.github.io
+ timestamp: TODAYS-DATE-YYYY-MM-DD
+ ---
```
Comment (hectorsector) [inline _pages/alex56dev.yaml]: @Alex56dev you'll need to fill in your details before being able to merge this. Start by replacing `YOUR-USERNAME` with your GitHub username.

File: _pages/alex56dev.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+ githubHandle: YOUR-USERNAME
+ pageUrl: YOUR-USERNAME.github.io
```
Comment (benweishi) [inline _pages/alex56dev.yaml]: Replace `YOUR-USERNAME.github.io` with your website created before. Like mine [benweishi.github.io/firstWebsite/](https://benweishi.github.io/firstWebsite/)

File: _pages/alex56dev.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+ githubHandle: YOUR-USERNAME
```
Comment (benweishi) [inline _pages/alex56dev.yaml]: Replace `YOUR-USERNAME` with your username at `github.com`.

File: _pages/alex56dev.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+ pageUrl: YOUR-USERNAME.github.io
+ timestamp: TODAYS-DATE-YYYY-MM-DD
```
Comment (benweishi) [inline _pages/alex56dev.yaml]: Replace `TODAYS-DATE-YYYY-MM-DD` with today's date, like `2017-03-25`.

**Comment by crichID** [comment]
@Alex56dev thank you for checking out our [on-demand classes](https://services.github.com/on-demand/). Since this file is not valid per the examples in the course, I am going to go ahead and close this Pull Request. If you would like to complete the activity, please feel free to re-open this Pull Request or start again. If you have any questions, please let us know!

**Pull request closed without merging**